app/main.py

```diff
@@ -16,4 +17,17 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+list_name_variables = [lucky_number, pi, one_is_a_prime_number,
+                       name, my_favourite_films, profile_info,
+                       marks, collection_of_coins]
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+
+for name_variable in list_name_variables:
```

Comment:
```suggestion
for variable in variables:
```

Reply:
I have problem.
if i change name_variable on variable and use flake8
.\app\[main.py:28](http://main.py:28/):5: VNE002 variable name 'variable' should be clarified.

i don't know how fix it.